packages/solana-connector/predicate/src/main.sw

```diff
@@ -38,13 +43,18 @@ pub fn b256_to_ascii_bytes(val: b256) -> Bytes {
 }
 
 configurable {
-    SIGNER: b256 = ZERO_B256,
+    SIGNER: b256 = b256::zero(),
 }
 
 fn main(witness_index: u64) -> bool {
-    let signature: B512 = tx_witness_data(witness_index).unwrap();
+    let witness_signature: B512 = tx_witness_data(witness_index).unwrap();
     let encoded = b256_to_ascii_bytes(tx_id());
-    let result = ed_verify(SIGNER, signature, encoded);
+
+    let signature = Signature::Ed25519(Ed25519::from(witness_signature));
+    let pub_key = PublicKey::from(SIGNER);
+    let message = Message::from(encoded);
+
+    let result = signature.verify(pub_key, message);
```

Comment:
The removal of `ed_verify` requires a double check.

Reply:
this can be fixed on another PR as we won't generate predicate versions now